src/m365/tenant/commands/report/report-settings-set.ts

```diff
@@ -0,0 +1,102 @@
+import { Logger } from '../../../../cli/Logger.js';
```

Comment:
A random comment about the necessary scope:

Let's update `pnp\cli-microsoft365\src\config.ts`: `https://graph.microsoft.com/Reports.Read.All` to `https://graph.microsoft.com/Reports.ReadWrite.All`